### PR TITLE
datapath: Add template substitutions for local conntrack maps.

### PR DESCRIPTION
**Pull request by jrajahalme**
When endpoint is configured with local conntrack maps, the dummy map
names in the template need to be substituted.

Fixes: d192b4c1f1b00ea0979e2f9282ab42d82d2f4e81
Signed-off-by: Jarno Rajahalme <jarno@covalent.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/7361)

### CODE REVIEW

**Comment by jrajahalme** [comment]
test-me-please

File: pkg/maps/ctmap/ctmap.go [hunk 1]
```diff
@@ -412,13 +412,14 @@ func maps(e CtEndpoint, ipv4, ipv6 bool) []*Map {
 			result = append(result, NewMap(MapNameAny6Global, MapTypeIPv6AnyGlobal))
 		}
 	} else {
+		idStr := fmt.Sprintf("%05d", int(e.GetID()))
```
Comment (joestringer) [inline pkg/maps/ctmap/ctmap.go]: nit: nit: `bpf.LocalMapName()` will add the names together with the proper "%05d", without duplicating the number of places where this `%05d` logic lives in the codebase.

**Review by joestringer — CHANGES_REQUESTED** [review]
Looks good, just one minor note for maintainability.

File: pkg/maps/ctmap/ctmap.go (hunk 1 above)
Comment (jrajahalme) [reply pkg/maps/ctmap/ctmap.go]: Updated!

**Comment by jrajahalme** [comment]
test-me-please

**Review by joestringer — APPROVED** [review]

**Comment by jrajahalme** [comment]
test-flannel

**Pull request merged**